### ISSUE TITLE
How can I send custom params to my events source?

### ISSUE THREAD

**Issue by viggyprabhu** [Serhioromano/bootstrap-calendar#695]
I want to send some custom params to the events source everytime it is hit to fetch data. One way I can understand is to send it as GET request params like 'events.json.php?id=100'. Is there some other better way to do this.

**Comment by eleazan** [comment]
Yeah, you can set ```events_source``` as a function, and fetch your own get/post / ajax call ;)
